apps/yggdrasil/lib/yggdrasil/user.ex

```diff
@@ -25,9 +27,22 @@ defmodule Yggdrasil.User do
     |> validate_length(:password, min: 4)
     |> validate_length(:password_confirmation, min: 4)
     |> validate_confirmation(:password)
+    |> default_role
```

Comment:
Yeah, maybe the changeset function should be something like `create_user_changeset` and then later `create_admin_changeset`. The user changset could auto assign a basic role and validate that the user wasn't given the admin role.